config/locales/en.yml

```diff
@@ -685,46 +685,46 @@ en:
           gondola: "Gondola Lift"
           magic_carpet: "Magic Carpet Lift"
           platter: "Platter Lift"
-          pylon: "Pylon"
+          pylon: "Aerialway Pylon"
           station: "Aerialway Station"
           t-bar: "T-Bar Lift"
           "yes": "Aerialway"
         aeroway:
-          aerodrome: "Aerodrome"
+          aerodrome: "Airport"
           airstrip: "Airstrip"
           apron: "Airport Apron"
           gate: "Airport Gate"
           hangar: "Hangar"
           helipad: "Helipad"
-          holding_position: "Holding Position"
+          holding_position: "Aircraft Holding Position"
           navigationaid: "Aviation Navigation Aid"
-          parking_position: "Parking Position"
+          parking_position: "Aircraft Parking Position"
           runway: "Runway"
           taxilane: "Taxilane"
           taxiway: "Taxiway"
           terminal: "Airport Terminal"
           windsock: "Windsock"
         amenity:
-          animal_boarding: "Animal Boarding"
+          animal_boarding: "Animal Boarding Facility"
           animal_shelter: "Animal Shelter"
-          arts_centre: "Arts Centre"
+          arts_centre: "Arts Center"
           atm: "ATM"
           bank: "Bank"
           bar: "Bar"
-          bbq: "BBQ"
+          bbq: "Barbecue"
```

Comment:
I think `BBQ` is just as clear, and it matches the tag.

Reply:
I was under the impression that “BBQ” only refers to a cookout or the dish, whereas the equipment would only be called a “barbecue” or a “grill”. But maybe this is too pedantic? Ironically, for the dish, `cuisine=bbq` has been deprecated in favor of `cuisine=barbecue`.

Reply:
Oh wow I've never considered that distinction before but you may be right. "Barbecue" may indeed be more appropriate for the grill itself. I do not have a strong preference here.